tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -271,64 +271,6 @@ def test_list_file_matches_multiple(self):
         ]
         assert matches == expected_matches
 
-    def test_normalized_stack_and_source_roots_starts_with_period_slash(self):
```

Comment:
See above comment about `find_roots`. We no longer need the `_normalized_stack_and_source_roots` function to normalize stack trace roots and source code roots because `find_roots` inherently does this.